### PR TITLE
Fields - hidden secrets

### PR DESCRIPTION
**Pull request by brianteeman**
If you want to search in the list of fields to find a specific field then there is more that you can do that simply search in the title. But there is no way to know this without reading the code. 

This PR adds the full information for searching for a field or field group as a tooltip. This is in exactly the same format as in other components.

This PR also alpha sorts the language file

image

image

If you want to check the search code you can find it 
https://github.com/joomla/joomla-cms/blob/25ba7e5097cc9ec34bab5dd9082f0247d7aa6cc2/administrator/components/com_fields/models/fields.php#L276-L295

https://github.com/joomla/joomla-cms/blob/25ba7e5097cc9ec34bab5dd9082f0247d7aa6cc2/administrator/components/com_fields/models/groups.php#L180-L192

### CODE REVIEW

File: administrator/components/com_fields/models/forms/filter_fields.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 			name="search"
 			type="text"
 			label=""
```
Comment (Quy) [inline administrator/components/com_fields/models/forms/filter_fields.xml:14]:
```suggestion
```

Comment (brianteeman) [reply administrator/components/com_fields/models/forms/filter_fields.xml:14]: out of scope of this PR

**Comment by brianteeman** [comment]
@zero-24 @SniperSister please can you look at the rips error

**Comment by SniperSister** [comment]
@brianteeman the drone.yml in your branch is outdated and incompatible to the new RIPS version that we are using. Updating the yaml (or updating the branch in general) will solve the issue.

**Comment by brianteeman** [comment]
Thanks

**Comment by brianteeman** [comment]
grrh - now it fails on some javascript that is unrelated to anything in this pr

**Comment by infograf768** [comment]
relaunched drone

**Review by laoneo — APPROVED** [review]

File: administrator/language/en-GB/en-GB.com_fields.ini [hunk 2]
```diff
@@ -7,12 +7,9 @@ COM_FIELDS="Fields"
 COM_FIELDS_BATCH_GROUP_LABEL="To Move or Copy your selection please select a group."
 COM_FIELDS_BATCH_GROUP_OPTION_NONE="- No Group -"
 COM_FIELDS_ERROR_UNIQUE_NAME="Another Field has the same name (remember it may be a trashed item or it may be already present as a custom field in another extension)."
+COM_FIELDS_FIELDS_FILTER_SEARCH_DESC="Search in field name, title or note. Prefix with ID: to search for a field ID. Prefix with Author: to search for a field author."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_fields.ini]: I would change it to be consistent with Articles search filter.

> Search in title, alias and note. Prefix with ID: or AUTHOR: or CONTENT: to search for an article ID, article author or search in article content.

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_fields.ini]: I thought about that but chose to go with th format used for all the other search tips.

Comment (Quy) [inline administrator/language/en-GB/en-GB.com_fields.ini]:
```suggestion
COM_FIELDS_FIELDS_FILTER_SEARCH_DESC="Search in field name, title or note. Prefix with ID: to search for a field ID. Prefix with AUTHOR: to search for a field author."
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f9e90d6dc19710814123c335c40f924ee9107cdf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26747">issues.joomla.org/tracker/joomla-cms/26747</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on f9e90d6dc19710814123c335c40f924ee9107cdf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26747">issues.joomla.org/tracker/joomla-cms/26747</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26747">issues.joomla.org/tracker/joomla-cms/26747</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**